inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/demo/send_demo.py

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import sys
+
+import inlong_dataproxy
+
+# user defined callback function
+def callback_func(inlong_group_id, inlong_stream_id, msg, msg_len, report_time, ip):
+    print("******this is call back, print info******")
+    print("inlong_group_id:", inlong_group_id, ", inlong_stream_id:", inlong_stream_id)
+    print("msg_len:", msg_len, ", msg content:", msg)
```

Comment:
The demo code should be as concise as possible, it is suggested to remove the log printing.

Reply:
This code is simple enough that I think it can be retained. It provides a definition for a simple callback function, informing users how to define the callback function and the meaning of each parameter in the `callback_func`.